src/pyhf/infer/utils.py

```diff
@@ -9,6 +9,22 @@
 log = logging.getLogger(__name__)
 
 
+def all_pois_floating(pdf, fixed_params):
+    """
+    Checks whether all POI(s) are floatig (i.e. not within the fixed set).
+
+    Args:
+        pdf (:obj:`pyhf.Model`): The model
+        fixed_params: array of bools
+
+    Returns:
+        bool: The result whether all POIs are floating.
```

Comment:
```suggestion
        :obj:`bool`: The result whether all POIs are floating.
```

Giordon started defaulting to using `obj` whenever possible, and I think that's a good habit to continue.